src/App.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import Home from './components/Home';
+import About from './components/About';
+import Contact from './components/Contact';
+import Navigation from './components/Navigation';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route 
+    exact path="/"
+    component={Home} />
+    <Route 
+    exact path="/about"
+    component={About} />
+    <Route 
+    exact path="/contact"
+    component={Contact} />
   </div>
```

Comment:
You don't need exact in all of the Routes, adding it to the index route `/` is enough